### PR TITLE
added commands for the stage 2 climb and mapped them to buttons

### PR DESCRIPTION
**Pull request by jboutin020106**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ClimbSubsystem.java [hunk 1]
```diff
@@ -8,6 +8,8 @@
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.sensors.CANCoderStatusFrame;
```
Comment (hazeltorek) [inline src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: what

**Review by hazeltorek — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -64,6 +73,9 @@ public RobotContainer() {
     toggleFieldOrientedBtn = new JoystickButton(driver, 5);
     toggleSlowModeBtn = new JoystickButton(driver, 7);
     deployButton = new JoystickButton(driver, 6);
+    stage2ForwardButton = new JoystickButton(driver, 4);
+    stage2BackwardButton = new JoystickButton(driver, 3);
```
Comment (JalenEvans) [inline src/main/java/frc/robot/RobotContainer.java]: Comment what buttons these are mapped to please :)

File: src/main/java/frc/robot/subsystems/ClimbSubsystem.java (hunk 1 above)
Comment (JalenEvans) [inline src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: If this doesn't do anything delete please :)

Comment (jboutin020106) [reply src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: I got rid of this just now

**Review by JalenEvans — COMMENTED** [review]
A couple changes need to be fixed before I approve 👍

**Review by JalenEvans — APPROVED** [review]
Changes were fixed. Less merge conflicts than before. I approve! :)
thumbs-up-192

**Review by hazeltorek — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -56,14 +64,18 @@ public RobotContainer() {
...
 
     //Buttons
-    toggleFieldOrientedBtn = new JoystickButton(driver, 5);
-    toggleSlowModeBtn = new JoystickButton(driver, 7);
-    deployButton = new JoystickButton(driver, 6);
+    toggleFieldOrientedBtn = new JoystickButton(driver, 5); // left trigger
+    toggleSlowModeBtn = new JoystickButton(driver, 7); // Select Button
+    deployButton = new JoystickButton(driver, 6); // right trigger
+    stage2ForwardButton = new JoystickButton(driver, 4); // Y button
+    stage2BackwardButton = new JoystickButton(driver, 3); // X button
```
Comment (hazeltorek) [inline src/main/java/frc/robot/RobotContainer.java]: create constants for these named `STAGE_TWO_FORWARD_BTN` and  `STAGE_TWO_BACKWARD_BTN`. the ones above it are already using constants as of #16

**Review by hazeltorek — CHANGES_REQUESTED** [review]

**Review by hazeltorek — APPROVED** [review]

**Pull request merged**